### ISSUE TITLE
Change version number shema of libfprintd ?

### ISSUE THREAD

**Issue by Vskilet** [NixOS/nixpkgs#75935]
**Describe the bug**
When I build nixos-unstable : 
```
[...]
checking for pkg-config... /nix/store/dx9m1whplkkhlch81z2k655sqpy53lij-pkg-config-0.29.2/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for libfprint > 0.1.0... no
configure: error: Package requirements (libfprint > 0.1.0) were not met:

No package 'libfprint' found
```

**To Reproduce**
1. git clone https://github.com/Vskilet/nixpkgs.git
2. git checkout nixos-unstable
3. sudo nixos-rebuild build -I nixpkgs=~/nixpkgs

**Expected behavior**
It seems that the version numbering scheme changed. How can I do to solve this dependency ?

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.86, NixOS, 20.03.git.b948cc4 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.1`
 - channels(root): `"nixos-20.03pre203904.bb1013511e1"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: fprintd-0.9.0
# a list of nixos modules affected by the problem
module: fprintd
```

**Comment by jtojnar** [comment]
We upgraded to libfprint 2.0 in #74656. What package is failing to build for you? It probably does not support libfprint 2 yet.

**Comment by Vskilet** [comment]
Ho yes you're right. I'll test this. I'll keep this open until the fix is merged

**Comment by Elyhaka** [comment]
Fixed by #75435
